### ISSUE TITLE
Graphs

### ISSUE THREAD

**Issue by lukealford** [bugdone/headshotbox#39]
So I've been thinking how we can make the player stats detailed. How about including some graphs for say KDR, WIN / LOSS on a monthly basis or weekly.

**Comment by bugdone** [comment]
Yeah, also maybe hs% for rifles/pistols.
Would be also interesting to have pie charts for win% for all matches and against players with bans.

**Comment by lukealford** [comment]
Yeah I was thinking the same, maybe have a the stats left align and on click of one of the stats it loads a the graph on the right side. then we are taking up the full with of the container rather than a small amount of data in the centre.

Lots of neat chart libraries available as well.

**Comment by cfulton** [comment]
Hi guys excellent work on what you have done.

One quick request for showing the MM chart, could you set it to change the interval on the x axis between number of rounds or time. 

Here is an example over 24 hours showing a huge gap between two sides.
http://imgur.com/28Ysuhc

Possibly add the button to switch as shown? Maybe not called 'Real time' rather just name 'Time' and 'Rounds' with a tool tip for X Axis.

**Comment by bugdone** [comment]
The rank changes only between games, so there's no point.
Your graph looks weird now only because it has too few data points. In a couple of days it will be more smooth.

**Comment by cfulton** [comment]
Sorry, I meant it should be matches. I updated the original post.

For example. 

![untitled](https://cloud.githubusercontent.com/assets/15142287/10516445/05fbf370-73a4-11e5-82c1-6409e736c5a9.png)

So that way when a user does not play for a long period of time, the graph won't show a long line to rank up. 

Don't remove the time graph to rank graph. But it would be nice to have a matches to rank graph.

**Comment by jcoester** [comment]
In Microsoft Excel i found this really interesting graph called "Waterfall" which could be really interesting in combination with the KD-Difference. 

I made an Example:
http://imgur.com/oQ62HjA

PS: I really love using headshotbox because i already was a big fan of Battelog (Battlefield Franchise). also using it since the first release (around April i think?). So keep the great work going! :)
